css/main.css

```diff
@@ -4,7 +4,10 @@
    Author's custom styles
    ========================================================================== */
 
-
+.poem-container {
+   width: 850px;
+   word-wrap: break-word;
```

Comment:
Having the word-wrap is an effective way to keep the flow of the page instead of using multiple break tags.